wp-graphql-smart-cache.php

```diff
@@ -273,3 +278,53 @@ function appsero_init_tracker_wpgraphql_smart_cache() {
 }
 
 appsero_init_tracker_wpgraphql_smart_cache();
+
+/**
+ * The callback function for saved query garbage collection event.
+ * Look for saved queries to cleanup and schedule a job to do small batches of deletes.
+ */
+add_action(
```

Comment:
This isn't a blocker now, just kind of noting as like food for thought, but I think we should consider moving some of this file into another "bootstrap" file of sorts. 

Keep it a bit more consistent with our other codebases that keep the main plugin file a bit more slim, i.e. https://github.com/wp-graphql/wp-graphql/blob/develop/wp-graphql.php